docs/cloud-edge/modules/user-agent-filter.md

```diff
@@ -61,26 +59,17 @@ func listenUserAgentFIlter(ctx context.Context) net.Listener {
-`(?:\/[A-Za-z0-9\.]+|) {0,5}([A-Za-z0-9 \-_\!\[\]:]{0,50}(?:[Aa]rchiver|[Ii]ndexer|[Ss]craper|[Bb]ot|[Ss]pider|[Cc]rawl[a-z]{0,50}))[/ ](\d+)(?:\.(\d+)(?:\.(\d+)|)|)`,  // Bots General matcher 'name/0.0'
-`(?:\/[A-Za-z0-9\.]+|) {0,5}([A-Za-z0-9 \-_\!\[\]:]{0,50}(?:[Aa]rchiver|[Ii]ndexer|[Ss]craper|[Bb]ot|[Ss]pider|[Cc]rawl[a-z]{0,50})) (\d+)(?:\.(\d+)(?:\.(\d+)|)|)`,     // Bots General matcher 'name 0.0'
-`((?:[A-z0-9]{1,50}|[A-z\-]{1,50} ?|)(?: the |)(?:[Ss][Pp][Ii][Dd][Ee][Rr]|[Ss]crape|[Cc][Rr][Aa][Ww][Ll])[A-z0-9]{0,50})(?:(?:[ /]| v)(\d+)(?:\.(\d+)|)(?:\.(\d+)|)|)`, // Bots containing spider|scrape|bot(but not CUBOT)|Crawl
-```
+Requests with multiple `user-agent` header’s will be matched as a single value.
+
+### Match Behavior
+
+Requests are allowed if they match an allow rule OR they match no deny rules. On deny requests are rejected with a `403 Forbidden` error.
```

Comment:
For the second sentence, consider rewording as "Requests matching deny rules are rejected with a `403 Forbidden` error" or "Denied requests are rejected with a `403 Forbidden` error".